app/controllers/dfe_sign_in_controller.rb

```diff
@@ -120,4 +120,11 @@ def choose_error_template
   def target_path_is_support_path
     @target_path&.match(/^#{support_interface_path}/)
   end
+
+  def user_ip_address
+    # If we are on AKS we need to use the x-real-ip header instead
+    # of the remote ip as X-FORWARDED-FOR constains the ip and proxies
+    # Rails is picking the proxy from last to first on remote_ip calls.
+    request.headers['x-real-ip'] || request.remote_ip
```

Comment:
Can `request.headers['x-real-ip']` ever be a truthy empty value like an empty string?

Reply:
Good question. No, but I can consider it in the code by calling `presence`.